core/lib/zksync_core/src/lib.rs

```diff
@@ -545,6 +545,12 @@ pub async fn initialize_components(
         tracing::info!("initialized State Keeper in {elapsed:?}");
     }
 
+    let diamond_proxy_addr = contracts_config.diamond_proxy_addr;
```

Comment:
nit - any reason why this is here and not in line 585?

Reply:
I guess no reason, but the name has changed, so I'll just keep as is